lib/motion-spec/specification.rb

```diff
@@ -101,7 +103,7 @@ def postpone_block_until_change(object_to_observe, key_path, timeout = 1, &block
       postponed_change_block_timeout_exceeded
     end
 
-    def observeValueForKeyPath(key_path, ofObject:object, change:_, context:__)
+    def observeValueForKeyPath(_key_path, ofObject:object, change:_, context:__)
```

Comment:
Unused method argument - `ofObject`. You can also write as `observeValueForKeyPath(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `change`. You can also write as `observeValueForKeyPath(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `context`. You can also write as `observeValueForKeyPath(*)` if you want the method to accept any arguments but don't care about them.